packages/imitation_game/README.md

```diff
@@ -0,0 +1,89 @@
+# Imitation Game
+
+## Description
+
+`imitation_game` is a platform for performing automated tests to compare the
+performance of different UI frameworks.  For example, how much memory does the
+same app written in Flutter and UIKit take?
+
+## Running all the tests
+
+You need a mobile device plugged into your computer and setup for development.
+The mobile device and the computer need to be on the same network, one that
+allows communication between computers since that's how the mobile phone will
+report its results to the computer.
+
+```sh
+dart imitation_game.dart
+```
+
+## Dependencies
+
+In order to run the tests you will need the union of all the platforms being
+tested.  As new tests are added please add to this list:
+
+### iOS
+
+- Flutter
+- Xcode
+- [ios_deploy](https://github.com/ios-control/ios-deploy) - used to launch apps
+  on the attached iOS device.
+
+## File Layout
+
+```
+./
+├─ imitation_game.dart
+└─ tests/
+   ├─ smiley/
+   │  ├─ README.md
+   │  ├─ flutter/
+   │  │  └─ run_ios.sh
+   │  └─ uikit/
+   │     └─ run_ios.sh
+   └─ memory/
+      ├─ README.md
+      ├─ flutter/
+      │  └─ run_ios.sh
+      └─ uikit/
+         └─ run_ios.sh
+```
+
+Here there are 2 different tests with 2 different platform implementations.  The
+tests are named `smiley` and `memory`, they are both implemented on the
```

Comment:
memory is self explanatory. Maybe smiley needs a more descriptive name? :D

Reply:
"Smiley" is just referencing the name in the example above.  It could have been anything.  What it is testing is ill-relevant to the point.  =)